dist/metacom.js

```diff
@@ -61,7 +61,7 @@ export class Metacom extends EventEmitter {
   createStream(name, size) {
     const streamId = ++this.streamId;
     const initData = { streamId, name, size };
-    const transport = this;
+    const transport = this.socket;
```

Comment:
GRASP: Information Expert principle violation. We have a wrapper around transport `WebsocketTransport` or `HttpTransport`, don't touch their internals, use just external interface.